traptor/traptor.py

```diff
@@ -153,6 +153,7 @@ def _setup_kafka(self):
                     'error_type': 'KafkaUnavailableError',
                     'ex': traceback.format_exc()
                 })
+                dd_monitoring.increment('traptor_error_occurred')
```

Comment:
We should use an `error_type` datadog tag so we can aggregate on the types of errors that occur. I'm not sure if the increment function allows for tag in this rendition of dd_monitoring.py, but you could pull it over from Cooper.